sdk/eventhub/event-hubs/src/eventPosition.ts

```diff
@@ -88,12 +94,12 @@ export class EventPosition {
   }
 
   /**
-   * Creates a position at the given offset.
-   * @param offset The offset of the data relative to the Event Hub partition stream.
-   * The offset is a marker or identifier for an event within the Event Hubs stream.
-   * The identifier is unique within a partition of the Event Hubs stream.
+   * Corresponds to the event in the partition at the provided offset.
```

Comment:
Corresponds to the "event" or "position"? Same for all other "from" methods

Reply:
Returns a position corresponding to the event in the partition at the provided offset?